src/luarocks/fs.lua

```diff
@@ -42,35 +42,4 @@ fs.verbose = function()    -- patch io.popen and os.execute to display commands
 end
 if cfg.verbose then fs.verbose() end
 
-local function load_fns(fs_table)
-   for name, fn in pairs(fs_table) do
-      if not fs[name] then
-         fs[name] = fn
-      end
-   end
-end
-
--- Load platform-specific functions
-local loaded_platform = nil
-for _, platform in ipairs(cfg.platforms) do
```

Comment:
Now we ignore config's OS autodetection. Commit message should explicitly state that, I think, because it is change of the behaviour.